diff --git a/src/feature.h b/src/feature.h
index 0c68f79f2..e35a2ad83 100644
--- a/src/feature.h
+++ b/src/feature.h
@@ -697,9 +697,9 @@
 /*
  * +termguicolors	'termguicolors' option.
  */
-#if (defined(FEAT_BIG) && defined(FEAT_SYN_HL)) && !defined(ALWAYS_USE_GUI)
+// #if (defined(FEAT_BIG) && defined(FEAT_SYN_HL)) && !defined(ALWAYS_USE_GUI)
 # define FEAT_TERMGUICOLORS
-#endif
+// #endif
 
 /*
  * +vartabs		'vartabstop' and 'varsofttabstop' options.
